Azure/azure-ad/ingest/parser.yml

```diff
@@ -58,10 +62,10 @@ stages:
           azuread.properties.appDisplayName: "{{parsed_event.message.properties.appDisplayName}}"
           azuread.properties.source: "{{parsed_event.message.properties.source}}"
           azuread.properties.detectionTimingType: "{{parsed_event.message.properties.detectionTimingType}}"
-          azuread.properties.status.errorCode: "{{parsed_event.message.properties.status.errorCode}}"
-          azuread.properties.status.additionalDetails: "{{parsed_event.message.properties.status.additionalDetails}}"
-          azuread.properties.status.failureReason: "{{parsed_event.message.properties.status.failureReason}}"
```

Comment:
Could you maintain `azuread.properties.status.errorCode` and `azuread.properties.status.failureReason`? These fields are currently used by customers.